### ISSUE TITLE
Potion Error

### ISSUE THREAD

**Issue by bobhenl** [GC-spigot/battle-pass#590]
### Describe the bug

Hello, I get this error in console.
```yml
15:24:49 ERROR]: Could not pass event PlayerItemConsumeEvent to BattlePass v4.6.9
java.lang.NullPointerException: Cannot invoke "org.bukkit.potion.PotionType.name()" because "<local8>" is null
        at BattlePass-4.6.9.jar/net.advancedplugins.bp.impl.actions.internal.ConsumeQuest.onItemConsume(ConsumeQuest.java:36) ~[BattlePass-4.6.9.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1348.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:1.21.1-2297-ef12ede]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:630) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.world.entity.LivingEntity.completeUsingItem(LivingEntity.java:4222) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.level.ServerPlayer.completeUsingItem(ServerPlayer.java:2079) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.world.entity.LivingEntity.updateUsingItem(LivingEntity.java:4079) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.level.ServerPlayer.updateUsingItem(ServerPlayer.java:2779) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.world.entity.LivingEntity.updatingUsingItem(LivingEntity.java:4059) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.world.entity.LivingEntity.tick(LivingEntity.java:3227) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.world.entity.player.Player.tick(Player.java:317) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.level.ServerPlayer.doTick(ServerPlayer.java:854) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.tick(ServerGamePacketListenerImpl.java:360) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.network.Connection.tick(Connection.java:647) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.network.ServerConnectionListener.tick(ServerConnectionListener.java:242) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1861) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:513) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1635) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1335) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:334) ~[purpur-1.21.1.jar:1.21.1-2297-ef12ede]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```
but not sure how to reproduce it

### How to reproduce

Unfortunately I¨m not aware

**Comment by KPGTB** [comment]
Hey! Fix will be available in one of the next updates

**Comment by bobhenl** [comment]
Still not fixed in 4.6.11
```yml
[06:59:18] [Server thread/ERROR]: Could not pass event PlayerItemConsumeEvent to BattlePass v4.6.11
java.lang.NullPointerException: Cannot invoke "org.bukkit.potion.PotionType.name()" because "<local8>" is null
	at BattlePass-4.6.11.jar/net.advancedplugins.bp.impl.actions.internal.ConsumeQuest.onItemConsume(ConsumeQuest.java:36) ~[BattlePass-4.6.11.jar:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor2098.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:1.21.1-2298-a61db94]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:630) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
	at net.minecraft.world.entity.LivingEntity.completeUsingItem(LivingEntity.java:4222) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.level.ServerPlayer.completeUsingItem(ServerPlayer.java:2079) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.world.entity.LivingEntity.updateUsingItem(LivingEntity.java:4079) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.level.ServerPlayer.updateUsingItem(ServerPlayer.java:2779) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.world.entity.LivingEntity.updatingUsingItem(LivingEntity.java:4059) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.world.entity.LivingEntity.tick(LivingEntity.java:3227) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.world.entity.player.Player.tick(Player.java:317) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.level.ServerPlayer.doTick(ServerPlayer.java:854) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.tick(ServerGamePacketListenerImpl.java:360) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.network.Connection.tick(Connection.java:647) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.network.ServerConnectionListener.tick(ServerConnectionListener.java:242) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1861) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:513) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1635) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1335) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:334) ~[purpur-1.21.1.jar:1.21.1-2298-a61db94]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```
